### PR TITLE
Add delivery:success and delivery:failure webhooks

### PR DESCRIPTION
**Pull request by dannytranlx**
I was thinking to maybe combine them, but not really sure how to handle the different trigger name and the lack of `error` on the successful one, so it's really explicit /shrug

@mspensieri @jugarrit @jpjoyal @lemieux @hebime

### CODE REVIEW

File: source/includes/rest/_core.md [hunk 1]
```diff
@@ -421,6 +421,8 @@ A webhook with a `postback` trigger will be fired every time a user clicks on [a
 | **message:appMaker**      | only messages with role `appMaker` or `whisper` |
 | **postback**              | when a user clicks on a postback action |
 | **merge:appUser**         | when two or more users are merged into one |
+| **delivery:success**      | when a delivery is successful |
```
Comment (mspensieri) [inline source/includes/rest/_core.md]: Might need to give some concept of what a delivery is, I don't think it's clear just from this that it means delivery to an OTT channel

**Comment by dannytranlx** [comment]
Thanks guys, will wait until we deploy the feature.

**Comment by gozman** [comment]
Can we merge this now?

**Comment by dannytranlx** [comment]
@jpjoyal wanted to update this feature before making it official (merging docs)

**Pull request merged**